src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -769,6 +769,8 @@ static sint32
     console.WriteLineError("OpenRCT2 build not compiled with Twitch integration.");
 #else
     // TODO: Add some twitch commands
+    // Display a message to the player for now
+    console.WriteLine("To be implemented");
```

Comment:
While not really part of this PR, I thought I'd add this message for now, to give players at least some message when they try the command out. This command is listed when typing "help" after all.